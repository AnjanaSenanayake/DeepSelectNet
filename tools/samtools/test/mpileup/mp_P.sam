@SQ	SN:z	LN:13
@CO	Various I combinations with (P)ads
@CO	AGCTT    AGCAG ref
@CO	AGCTTGGCCAGCAG
@CO	AGCTTGG  AGCAG
@CO	AGCTT GC AGCAG
@CO	AGCTT  CCAGCAG
@CO	01234ABCD56789 qual
s1	0	z	2	0	10M	*	0	0	AGCTTAGCAG	0123456789
s1	0	z	2	0	5M4I5M	*	0	0	AGCTTGGCCAGCAG	01234ABCD56789
s1	0	z	2	0	5M2I2P5M	*	0	0	AGCTTGGAGCAG	01234AB56789
s1	0	z	2	0	5M1P2I1P5M	*	0	0	AGCTTGCAGCAG	01234BC56789
s1	0	z	2	0	5M2P2I5M	*	0	0	AGCTTCCAGCAG	01234CD56789
