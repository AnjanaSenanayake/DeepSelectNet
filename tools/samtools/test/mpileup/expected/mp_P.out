z	2	A	5	^!.^!.^!.^!.^!.	00000
z	3	G	5	.....	11111
z	4	C	5	.....	22222
z	5	T	5	.....	33333
z	6	T	5	..+4GGCC.+4GG**.+4*GC*.+4**CC	44444
z	7	A	5	.....	55555
z	8	G	5	.....	66666
z	9	C	5	.....	77777
z	10	A	5	.....	88888
z	11	G	5	.$.$.$.$.$	99999
